lib/matplotlib/collections.py

```diff
@@ -1985,15 +1985,26 @@ def __init__(self, coordinates, *, antialiased=True, shading='flat',
 
         _api.check_shape((None, None, 2), coordinates=coordinates)
         self._coordinates = coordinates
-        self._antialiased = antialiased
         self._shading = shading
         self._bbox = transforms.Bbox.unit()
         self._bbox.update_from_data_xy(self._coordinates.reshape(-1, 2))
+        kwargs["antialiased"] = antialiased
```

Comment:
```suggestion
```
I think you can remove this, and remove antialiased from the signature above and then it is in `kwargs` already.